packages/config/config/devices/0x010f/fgcd-001.json

```diff
@@ -140,40 +166,37 @@
 				}
 			]
 		},
-		"7": {
-			"label": "Associations in Z-Wave network security mode",
-			"description": "Secure or non-secure commands to assoc groups",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 63,
-			"defaultValue": 63,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
+		"7[0x01]": {
+			"$import": "templates/fibarogroup_templates.json#secure_assoc_bit",
+			"label": "Associations: Send secure commands to 2nd group"
 		},
-		"10": {
-			"label": "Commands sent to 2nd association group (CO Alarm)",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 3,
-			"defaultValue": 3,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "BASIC ON",
-					"value": 1
-				},
-				{
-					"label": "BASIC OFF",
-					"value": 2
-				},
-				{
-					"label": "BASIC ON & BASIC OFF",
-					"value": 3
-				}
-			]
+		"7[0x02]": {
+			"$import": "templates/fibarogroup_templates.json#secure_assoc_bit",
+			"label": "Associations: Send secure commands to 3rd group"
+		},
+		"7[0x04]": {
+			"$import": "templates/fibarogroup_templates.json#secure_assoc_bit",
+			"label": "Associations: Send secure commands to 4th group"
+		},
+		"7[0x08]": {
+			"$import": "templates/fibarogroup_templates.json#secure_assoc_bit",
+			"label": "Associations: Send secure commands to 5th group"
+		},
+		"7[0x0f]": {
+			"$import": "templates/fibarogroup_templates.json#secure_assoc_bit",
+			"label": "Associations: Send secure commands to 6th group"
+		},
```

Comment:
That's not correct. 0x0f is not a single bit, it selects the lower 4 bits.
This also means the follwing bitmask is incorrect.